test/index.js

```diff
@@ -6,69 +6,53 @@ const Code = require('code');
 const Hapi = require('hapi');
 const Lab = require('lab');
 const Redis = require('../');
-
-// Declare internals
-
-const internals = {};
-
-
-// Test shortcuts
-
 const lab = exports.lab = Lab.script();
 const it = lab.it;
 const expect = Code.expect;
 
-
```

Comment:
[Two empty lines between module functions or assignments](https://github.com/hapijs/contrib/blob/master/Style.md#newlines)  (end of function to comment about next function)